lib/testing/testDataCheck.hpp

```diff
@@ -141,7 +141,7 @@ void testDataCheck<A_Type>::main_thread() {
             }
         }
 
-        if (num_errors > 0) {
+        if (num_errors == 0) {
```

Comment:
That change looks like a bug.

Reply:
I think it was a bug before. If there are no errors, print out some info. Otherwise, fail the test.